### PR TITLE
Using stdlib for python version checks

### PR DESCRIPTION
**Pull request by kchawla-pi**
- Less code coupling

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
>   • M nilearn/tests/test_py2_deprecation_warning.py (6)

Also, could you rename test_py2_deprecation_warning.py to test_init.py,
as the convention is to have the tests for one file in a file with the
same name prefixed by "test_".

File: nilearn/__init__.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def _py2_deprecation_warning():
     warnings.simplefilter('once')
     py2_warning = ('Python2 support is deprecated and will be removed in '
                    'a future release. Consider switching to Python3.')
-    if six.PY2:
+    if LooseVersion(sys.version) < LooseVersion('3'):
```
Comment (GaelVaroquaux) [inline nilearn/__init__.py]: The best way to do this is to use sys.version_info. It gives safer and simpler comparisons.

Comment (kchawla-pi) [reply nilearn/__init__.py]: Cool! I don't even need LooseVersion. Thanks.

**Comment by bthirion** [comment]
LGTM.

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thank you

**Pull request merged**